src/Components/Carousel/Carousel.js

```diff
@@ -61,17 +94,25 @@ const Carousel = () => {
                     <img src={e.hdurl} className="ImageWrapper"/>
```

Comment:
img elements must have an alt prop, either with meaningful text, or an empty string for decorative images. You can use e.title or even empty string is fine.